### PR TITLE
Change ticket length for Autoservicio

### PR DESCRIPTION
**Pull request by DarkFrostnight**
In the new version of the documentation the ticket has a length of 20

### CODE REVIEW

**Review by isabellaahc — DISMISSED** [review]
LGTM

File: TransbankPosSDK/POSAutoservicio.cs [hunk 1]
```diff
@@ -102,9 +102,9 @@ public async Task<SaleResponse> Sale(int amount, string ticket, bool sendVoucher
             {
                 throw new TransbankSaleException("Amount must be less than 999999999.");
             }
-            if (ticket.Length != 6)
+            if (ticket.Length > 20)
             {
-                throw new TransbankSaleException("Ticket must be 6 characters.");
+                throw new TransbankSaleException("The ticket must up to 20 characters.");
```
Comment (gonzunigad) [inline TransbankPosSDK/POSAutoservicio.cs:107]:
```suggestion
                throw new TransbankSaleException("The ticket must be up to 20 characters.");
```

File: TransbankPosSDK/POSAutoservicio.cs [hunk 2]
```diff
@@ -128,9 +128,9 @@ public async Task<MultiCodeSaleResponse> MultiCodeSale(int amount, string ticket
             {
                 throw new TransbankMultiCodeSaleException("Amount must be less than 999999999.");
             }
-            if (ticket.Length != 6)
+            if (ticket.Length > 20)
             {
-                throw new TransbankMultiCodeSaleException("Ticket must be 6 characters.");
+                throw new TransbankMultiCodeSaleException("The ticket must up to 20 characters.");
```
Comment (gonzunigad) [inline TransbankPosSDK/POSAutoservicio.cs:133]:
```suggestion
                throw new TransbankMultiCodeSaleException("The ticket must be up to 20 characters.");
```

**Review by gonzunigad — APPROVED** [review]

**Review by isabellaahc — APPROVED** [review]
👌

**Pull request merged**